main.py

```diff
@@ -2,6 +2,40 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = [
+        ("Western",),
+        ("Action",),
+        ("Dramma",),
```

Comment:
There is no need to add genres in tuples and after that index them during retrieval, place them as strings inside of a list